### ISSUE TITLE
Blank plot in html using nbconvert (basic template)

### ISSUE THREAD

**Issue by ltetrel** [plotly/plotly.py#1033]
Hi everyone,

I know that to convert a jupyter notebook (.ipynb) to an html I can use this command :

`jupyter nbconvert --to html --template full myNotebook.ipynb`

In this case I can successfully see my plot in the resulting html (myNotebook_full.html in 
[myNotebook.zip](https://github.com/plotly/plotly.py/files/2103910/myNotebook.zip)).

However, when I am trying to convert it using the basic template, there is a blank space instead of the plot 

(myNotebook_basic.html in [myNotebook.zip](https://github.com/plotly/plotly.py/files/2103910/myNotebook.zip)).

`jupyter nbconvert --to html --template basic myNotebook.ipynb`

![missing](https://user-images.githubusercontent.com/37963074/41436904-1b0388e0-6ff1-11e8-8fa3-8f3097977109.png)

Do you have an idea why ?

Thanks all for your work and support !

**Comment by ltetrel** [comment]
I finally managed to resolve my issue.
I just needed to include the following scripts at the top of my .html template file :

`<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>`

**Comment by Juanlu001** [comment]
For Sphinx docs that use themes without jQuery on `<head>`, this workaround is fragile and does not always work in all browsers or all the times. We are tracking it accross several places:

spatialaudio/nbsphinx#128, poliastro/poliastro#281, rtfd/readthedocs.org#4367

Perhaps this is something that could be worked on the Plotly side? Could we please reopen this?

**Comment by jonmmease** [comment]
Hi @Juanlu001, sure we can reopen this.

So it looks to me like the JavaScript block emitted by plotly.py on `init_notebook_mode` expects `requirejs` to be active, but I don't see how it's using or altering jquery.

https://github.com/plotly/plotly.py/blob/1e4a3c7e1cfc5a8e40a847e7242ed54b20a9050b/plotly/offline/offline.py#L129-L141

Could you summarize your current understanding of what's going wrong?

Another approach would be to use the fact the `FigureWidget` is an ipywidget.  I've successfully embedded `FigureWidget` instances for nbviewer, and it's supposed to be possible in sphinx as well (https://ipywidgets.readthedocs.io/en/stable/embedding.html) though I haven't tried that yet.

**Comment by Juanlu001** [comment]
> Could you summarize your current understanding of what's going wrong?

Yes! Please read my comment here:

https://github.com/rtfd/readthedocs.org/issues/4367#issuecomment-423814712

**Comment by jonmmease** [comment]
Hi @Juanlu001, just following up. I haven't studied the thread in detail, but it seems like folks are making progress on the RTD side.  Is there anything you see at this point that you need from plotly.py?

**Comment by Juanlu001** [comment]
@jonmmease Thanks! Progress is slow but I think it's been acknowledged that it's an issue on the RTD side - let's hope someone gives it the final push.

**Comment by gvwilson** [comment]
Hi - we are currently trying to tidy up Plotly's public repositories to help us focus our efforts on things that will help users most. Since this issue has been sitting for several years, so I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our backlog. Thanks for your help - @gvwilson
